apps/docs/src/components/code/ExpandableLayout.tsx

```diff
@@ -41,18 +42,27 @@ export const ExpandableLayout = ({ scope, code }: ExpandableLayoutProps) => {
       />
 
       {/* Preview Section */}
-      <div
-        className={clsx(
-          "p-md py-[40px] bg-bgPage border rounded-inherit",
-          isExpanded && "rounded-b-0",
-        )}
-      >
-        {liveError ? (
-          <div className="text-negative">{liveError}</div>
-        ) : (
-          <div>{element}</div>
-        )}
-      </div>
+      <DocsProvider>
+        <div
+          className={clsx(
+            "p-md py-[40px] bg-bgPage border rounded-inherit",
+            isExpanded && "rounded-b-0",
+          )}
+        >
+          <div
+            className={clsx(
+              "p-md pt-lg bg-bgPage border rounded-inherit",
+              isExpanded && "rounded-b-0",
+            )}
+          >
```

Comment:
this shouldn't be here - probably issues with the rebase
![image](https://github.com/user-attachments/assets/7b322cb7-31a5-42d3-94d2-50ed7d2f310f)
![image](https://github.com/user-attachments/assets/57673877-7413-4f2b-b4c1-f9e3cf24adad)